pygitguardian/sca_models.py

```diff
@@ -13,12 +13,14 @@ class SCAIgnoredVulnerability(Base, FromDictMixin):
     """
     A model of an ignored vulnerability for SCA. This allows to ignore all occurrences
     of a given vulnerability in a given dependency file.
-    - identifier: identifier (currently: GHSA id) of the vulnerability to ignore
+    - identifier: OSV id (currently: only GHSA id) or CVE id of the vulnerability to ignore
+    - identifier_type: type of the identifier (OSV id or CVE id)
     - path: the path to the file in which ignore the vulnerability
     """
 
-    identifier: str
     path: str
+    identifier: str
+    identifier_type: Literal["osv", "cve"] = "osv"
```

Comment:
It would be nice if this could be an enum, unless this causes issues.

Reply:
The identifier type field will probably end up not being necessary. I am not closing this MR because I will use it to update comments as the identifier type will be allowed to be a CVE id.